changelog.md

```diff
@@ -0,0 +1,101 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [1.3.0] - 2022-05-26
+### Added
+- Changelog file
+- Pyramid middleware
+- CherryPy middleware
+- FastAPI middleware
+
+## [1.2.0] - 2022-05-11
+### Added
+- Bottle middleware
+- Bottle sample example
+
+### Fixed
+- Errors filtered out when middleware involve in backtrace [#191](https://github.com/airbrake/pybrake/issues/191)
+
+## [1.1.0] - 2022-05-09
+### Changed
+- Package Restructure
+- Add Django sample example
+- Update Flask sample example
+
+### Fixed
+- Fix query stats send to the Airbrake
+
+## [1.0.6] - 2022-04-13
+### Changed
+- Improve pytest
+
+### Fixed
+- [#183](https://github.com/airbrake/pybrake/issues/183) on Flask request filter
```

Comment:
This looks confusing and not useful if read this changelog anywhere else but GitHub. Let's describe #183 here.